spec/support/vcr.rb

```diff
@@ -18,6 +18,13 @@
       }
     ]
   }
+
+  # If the environment variable DROPBOX_OAUTH_BEARER is present, its value
+  # appearing inside any cassette will automatically be replaced with the
+  # string "MOCK_AUTHORIZATION_BEARER".
+  config.filter_sensitive_data('MOCK_AUTHORIZATION_BEARER') do
+    ENV['DROPBOX_OAUTH_BEARER']
+  end
```

Comment:
I can see there's a Rake task for this (`rake vcr:mock_authorization_bearer`).

I'm not sure if you're aware VCR can handle this for you automatically as you run the tests, but I've included it here as I found it useful while testing.